frontends/mit-open/src/page-components/SearchDisplay/SearchDisplay.tsx

```diff
@@ -503,6 +503,7 @@ const SearchDisplay: React.FC<SearchDisplayProps> = ({
   setSearchParams,
 }) => {
   const [searchParams] = useSearchParams()
+  const scrollHook = useRef<HTMLInputElement>(null)
```

Comment:
I think this should just be `HTMLElement` because it's a `div`

Reply:
Good spot, thanks.